### PR TITLE
Final changes

### PR DESCRIPTION
**Pull request by rachelbock**
Modified New User and Update User Dialog fragment to take in a B. Added height inch and foot check.

### CODE REVIEW

File: app/src/main/java/com/rage/clamber/Fragments/HomeActivity/UserInfo/UpdateUserDialogFragment.java [hunk 1]
```diff
@@ -92,4 +100,26 @@ public void onClick(DialogInterface dialog, int which) {
     public boolean isEditTextEmpty(EditText editText){
         return editText.getText().toString().trim().length() == 0;
     }
+
+    /**
+     * Method to determine whether the user input skill level is valid.
+     * @param skillLevel - the user intput string
```
Comment (nonvirtualthunk) [inline app/src/main/java/com/rage/clamber/Fragments/HomeActivity/UserInfo/UpdateUserDialogFragment.java]: intput -> input

File: app/src/main/java/com/rage/clamber/Fragments/LoginActivity/NewUserDialogFragment.java [hunk 2]
```diff
@@ -83,8 +90,28 @@ public void onClick(DialogInterface dialog, int which) {
...
+    protected boolean isEditTextEmpty(EditText editText) {
         return editText.getText().toString().trim().length() == 0;
     }
+    /**
+     * Method to determine whether the user input skill level is valid.
+     * @param skillLevel - the user intput string
+     * @return - an integer value that will be stored in the database. If the string is not valid
+     * it returns a defined invalid int to check for.
+     */
+    protected int getSkillLevel(String skillLevel) {
```
Comment (nonvirtualthunk) [inline app/src/main/java/com/rage/clamber/Fragments/LoginActivity/NewUserDialogFragment.java]: Could this be made into a `public static` method somewhere? That way you could call it from both places that use it without having to repeat the code.

**Comment by nonvirtualthunk** [comment]
+1, pending unification of the two `getSkillLevel` functions

File: app/src/main/java/com/rage/clamber/Fragments/LoginActivity/NewUserDialogFragment.java (hunk 2 above)
Comment (rachelbock) [inline app/src/main/java/com/rage/clamber/Fragments/LoginActivity/NewUserDialogFragment.java]: Yep - good idea.

File: app/src/main/java/com/rage/clamber/Fragments/HomeActivity/UserInfo/UpdateUserDialogFragment.java (hunk 1 above)
Comment (rachelbock) [inline app/src/main/java/com/rage/clamber/Fragments/HomeActivity/UserInfo/UpdateUserDialogFragment.java]: thanks!

**Comment by nonvirtualthunk** [comment]
Looks good, merging

**Pull request merged**